lib/dirList.js

```diff
@@ -149,7 +149,7 @@ const dirList = {
       route = path.normalize(path.join(route, '..'))
```

Comment:
What if we change it to?

Maybe on top of the file we do:
const pathPosixNormalize = path.posix.normalize

and then do:

```suggestion
      route = path.posix.normalize(path.join(route, '..'))
```

Reply:
This might work but it's not part of the fix for this PR. See https://github.com/fastify/fastify-static/pull/452#discussion_r1587438430